src/coreclr/tools/aot/ObjWriter/objwriter.h

```diff
@@ -15,6 +15,12 @@
 #include "llvm/DebugInfo/CodeView/TypeTableBuilder.h"
 
 #include "cfi.h"
+// This constant should be added to CFI_OPCODE enumeration in cfi.h
```

Comment:
You can just add it there (in this repo). 

We are doing integration of general goodness fixes from dotnet/runtimelab to dotnet/runtime every once in a while, and this can get easily included in the batch.